thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/template.hbs

```diff
@@ -7,21 +7,12 @@
             <span class="rootcause-anomaly__title">Is this a true anomaly?</span>
           </div>
         {{/if}}
-        <div class="col-xs-2">
-          {{#power-select
-            options=options
-            selected=status
-            onchange=(action "onFeedback")
-          as |option|}}
-            {{option}}
-          {{/power-select}}
-        </div>
-        <div class="col-xs-5">
+        <div class="col-xs-7">
           <span class="rootcause-anomaly__note"><a href="https://iwww.corp.linkedin.com/wiki/cf/display/ENGS/Anomaly+Labeling+on+ThirdEye#AnomalyLabelingonThirdEye-WhataretheLabels?" target="_blank">Learn more about each option</a></span>
         </div>
-        <div class="col rootcause-anomaly--icon">
-          <i class="glyphicon {{if isHidden " glyphicon-menu-up " "glyphicon-menu-down "}}" {{action "toggleHidden"}}></i>
-        </div>
+      </div>
+      <div class="col rootcause-anomaly__icon">
```

Comment:
I know it's copy-pasta, but what is this "col" class here?

Reply:
It doesn't do anything. I'll remove it. Thanks for the catch.